Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -931,9 +992,10 @@ def makeAllGlyphClassDefinitions(kerningPerScript, context, feaFile=None):
 
     # Generate common class names first so that common classes are correctly
     # named in other lookups.
-    if COMMON_SCRIPT in kerningPerScript:
-        common_pairs = kerningPerScript[COMMON_SCRIPT]
-        for pair in common_pairs:
+    for scripts, pairs in kerningPerScript.items():
+        if set(scripts) != COMMON_SCRIPTS_SET:
+            continue
```

Comment:
I believe the for loop is redundant because the `kerningPerScript` dict only contains one item (if at all) keyed `COMMON_SCRIPTS_SET`, so just like before you can test `if COMMON_SCRIPTS_SET in kerningPerScript: ...`

Reply:
`COMMON_SCRIPTS_SET` is a set and can’t be used as dict key.